README.md

```diff
@@ -29,61 +29,31 @@ Required env variables:
 - `WORKSPACE=dev`: env for local development, set by Terraform in AWS environments.
 - `SENTRY_DSN`: only needed in production.
 
-### To verify local changes in Dev1
+### Integration Tests for Dev1
 
-- Ensure your aws cli is configured with credentials for the Dev1 account.
-- Ensure you have the above env variables set in your .env, matching those in our Dev1 environment.
-- Add the following to your .env: `LAMBDA_FUNCTION_URL=<the Dev1 lambda function URL>`
-- Publish the lambda function:
+Some minimal integration tests are provided for checking the deployed webhook handling lambda, defined as `integration` type pytest tests.  These tests check the following:
+  * lambda function URL is operational
+  * `GET` and `POST` requests are received
+  * deployed lambda has adequate permissions to communicate with S3, StepFunctions, etc.
 
-  ```bash
-  make publish-dev
-  make update-lambda-dev
-  ```
+Other notes about tests:
+  * tests are limited to `Dev1` environment
+  * AWS `AWSAdministratorAccess` role credentials must be set on developer machine
+  * environment variable `WORKSPACE=dev` must be set
+    * no other environment variables are required, these are all retrieved from deployed context
 
-#### GET request example
+#### Steps to run integration tests
```

Comment:
Can we use numbering to enumerate these steps? I think it would help with the readability! :)